services/user/Permissions/plugin/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "permissions"
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+publish = false
```

Comment:
This should be `publish.workspace` now that `publish` is part of the workspace package.